src/controllers/route.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const path = require('path');
+const router = express.Router();
+
+const home = require('../views/home');
+
+router.get('/', (req, res) => {
+   res.render('home', { activePage: {home: true}});
+});
+
```

Comment:
We were adding codes in another branch already but we've added a fix to the branch called 'headerPartial'